backend/src/test/java/de/neuefische/readandmeet/backend/controller/MeetingControllerTest.java

```diff
@@ -47,4 +48,29 @@ void expectMeetingList_whenGETMeetingList() throws Exception {
                 //THEN
                 .andExpect(MockMvcResultMatchers.content().json(expected)).andExpect(MockMvcResultMatchers.status().isOk());
     }
+
+    @Test
+    @DirtiesContext
+    void expectUpdatedMeetingList_whenPOSTNewMeeting() throws Exception {
+        // GIVEN
+        String meetingWithoutId = """
+                   {
+                   "title": "book",
+                   "date": "2023-08-08",
+                   "location": "online"
+                   }
+                                """;
+
+
+        //WHEN
+        mockMvc.perform(MockMvcRequestBuilders.post("/api/meetings").content(meetingWithoutId).contentType(MediaType.APPLICATION_JSON))
+
+        //THEN
+                .andExpect(MockMvcResultMatchers.status().isOk())
+                .andExpect(MockMvcResultMatchers.content().contentType(MediaType.APPLICATION_JSON))
+                .andExpect(MockMvcResultMatchers.jsonPath("$[0].id").isNotEmpty())
+                .andExpect(MockMvcResultMatchers.jsonPath("$[0].title").value("book"))
+                .andExpect(MockMvcResultMatchers.jsonPath("$[0].date").value("2023-08-08"))
+                .andExpect(MockMvcResultMatchers.jsonPath("$[0].location").value("online"));
+    }
 }
```

Comment:
a blank line seems to be missing here as well 👀

Reply:
hups 🙃.. thank you adlerauge 🦅